### PR TITLE
Support free plan to query

### PR DESCRIPTION
**Pull request by neo-sun**

### CODE REVIEW

File: proxy/src/server.rs [hunk 1]
```diff
@@ -80,7 +80,10 @@ pub async fn start_server(host: &str, port: u16) {
     axum::Server::bind(&addr).serve(app.into_make_service()).await.unwrap();
```
Comment (HZHAIX) [inline proxy/src/server.rs:80]: May need change  `app.into_make_service()` to `app.into_make_service_with_connect_info::<MyConnectInfo>())`  to get connection information successfully

File: proxy/src/auth.rs [hunk 2]
```diff
@@ -72,14 +73,18 @@ pub async fn create_jwt(payload: Payload, daily: u64, rate: u64) -> Result<Strin
...
         deployment_id: payload.deployment_id,
         iat: payload.timestamp,
         exp: expiration,
     };
 
+    if let Some(addr) = free {
+        claims.agreement = Some(format!("{}", addr.ip()));
+    }
+
     if let Some(agreement) = payload.agreement {
```
Comment (HZHAIX) [inline proxy/src/auth.rs:88]: Should get from `claims.agreement` other than `payload.agreement`?

File: proxy/src/server.rs (hunk 1 above)
Comment (neo-sun) [reply proxy/src/server.rs]: fixed

File: proxy/src/auth.rs (hunk 2 above)
Comment (neo-sun) [reply proxy/src/auth.rs]: fixed

**Review by HZHAIX — APPROVED** [review]

**Pull request merged**